qa/rpc-tests/test_framework/util.py

```diff
@@ -215,6 +218,9 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     if binary is None:
         binary = os.getenv("BITCOIND", "bitcoind")
     args = [ binary, "-datadir="+datadir, "-keypool=1", "-discover=0", "-rest" ]
+    if os.getenv("QT", ""):
```

Comment:
Right. I used the same way than PYTHON_DEBUG is implemented. But i agree, maybe we should change both variables so that they only trigger if they are set with a non-0 value.